content/tools/aws-vault.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: AWS Vault
-description: "The `aws-vault` is a command line tool for securely storing and accessing encrypted AWS credentials for local development environments. It makes it extremely easy to work with IAM assumed roles across multiple AWS organizations."
+description: "`aws-vault` is a command line tool for securely storing and accessing encrypted AWS credentials for local development environments. It makes it extremely easy to work with IAM assumed roles across multiple AWS organizations."
```

Comment:
I'm torn. I don't like starting a sentence with just the lower-case command name. I'm not a grammarian, but doesn't feel right. @aknysh any suggestions?

Reply:
Perhaps we should just reword so that we can say `The` =)

```
The `aws-vault` command line tool is a utility for securely storing and accessing encrypted AWS credentials for local development environments.
```